.php_cs.dist

```diff
@@ -6,6 +6,8 @@ $config
     ->setUsingCache(false)
     ->getFinder()
     ->in(__DIR__)
-    ->exclude('vendor');
+    ->exclude('vendor', 'tests/integration/module-samples', 'tests/integration/expected')
+    ->exclude('tests/integration/module-samples')
+    ->exclude('tests/integration/expected');
```

Comment:
`exclude()` handle an array of path or a single string. As your line will not be taken in account, we should have

```suggestion
    ->exclude(['vendor', 'tests/integration/module-samples', 'tests/integration/expected']);
```

or

```suggestion
    ->exclude('vendor')
    ->exclude('tests/integration/module-samples')
    ->exclude('tests/integration/expected');
```

Reply:
I chose the 1st one https://github.com/PrestaShopCorp/header-stamp/pull/23/commits/559b92b59b8b97fb6327c3afc4524274cf1c5fed